src/nexpy/gui/pyqt.py

```diff
@@ -1,8 +1,10 @@
+import matplotlib
 import sip
 for api in ['QString', 'QVariant']:
     sip.setapi(api, 2)
 
-from qtconsole.qt import QtCore, QtGui, QtSvg, QT_API
+matplotlib.use('Qt4Agg')
```

Comment:
I would do this by importing from qtconsole, but then ask it which version of Qt you have and then set mpl to be what ever it needs to be.